src/components/about/contact-person.tsx

```diff
@@ -22,61 +20,65 @@ export const ContactPerson: React.FC<ContactPersonProps> = ({
   linkedIn,
 }) => {
   return (
-    <Flex sx={{ flexDirection: 'column' }}>
-      <Box sx={{ width: ['100%', '15rem'] }}>
+    <Flex sx={{ flexDirection: 'column', alignSelf:'center' }}>
+      <Box sx={{ width: '25rem', maxWidth: '25rem' }}>
         <AspectRatio
-          ratio={308 / 440}
+          ratio={459 / 527}
           sx={{ borderRadius: 6, overflow: 'hidden' }}
         >
           <Image
             src={photoPath}
             sx={{
               minWidth: '100%',
               minHeight: '100%',
-              width: 'auto',
-              height: 'auto',
+              width: '459px',
+              height: '527px',
```

Comment:
```suggestion
              maxWidth: '459px',
```

Maybe this can be changed to an aspect ratio?

Reply:
Funka med maxHeight!